test/test_nicolet_reading.m

```diff
@@ -0,0 +1,150 @@
+function test_nicolet_reading
+
+% MEM 2gb
+% WALLTIME 00:30:00
+% DEPENDENCY test_nicolet_reading_onefile read_nervus_header
+
+% function to test reading of Nicolet/Nervus EEG files
+% one is a file from 2006 (older Nicolet format)
+% one is a file from 2018 (newer Nicolet format)
+
+%path_to_load = dccnpath('/home/common/matlab/fieldtrip/data/test/original/eeg/nicolet');
+path_to_load = 'C:\Midlertidig_Lagring\FieldTripNicoletTestData';
```

Comment:
It would be great if we could get a version of this testdata, so that we can store it on our filesystem.

Reply:
Sorry, forget about this comment, I guess these are the data that you point to in your own github repo...

Reply:
These data are available in https://github.com/janbrogger/FieldTripNicoletTestData

Reply:
Yes indeed, thanks for inviting me to the repo